broker/src/main/java/io/camunda/zeebe/broker/Broker.java

```diff
@@ -428,4 +445,16 @@ public PartitionManager getPartitionManager() {
   private static final class TestCompanionClass {
     private AtomixCluster atomix;
   }
+
+  private static class BrokerStartupActor extends Actor {
+
+    @Override
+    public String getName() {
+      return "Broker Startup Actor";
+    }
+
+    public <V, T extends ActorFuture<V>> ActorFuture<V> call(final Callable<T> callable) {
+      return actor.call(callable).join();
```

Comment:
:thought_balloon: I understand why it's here, but since it's also called `call`, I expected asynchronous semantics like with the normal actor stuff, and blocking here is definitely unexpected for me.